components/ntp_background_images/browser/ntp_background_images_service.cc

```diff
@@ -131,20 +133,43 @@ void NTPBackgroundImagesService::Init() {
       DVLOG(2) << __func__ << ": NTP SR test data will be loaded"
                << " from local path at: "
                << forced_local_path.LossyDisplayName();
-      OnComponentReady(false, forced_local_path);
+      OnComponentReady(false, true, forced_local_path);
     } else {
       CheckSuperReferralComponent();
     }
   }
 #endif
 }
 
-void NTPBackgroundImagesService::CheckSIComponentUpdate(
+void NTPBackgroundImagesService::CheckImagesComponentUpdate(
     const std::string& component_id) {
-  DVLOG(2) << __func__ << ": Check NTP SI component update";
+  DVLOG(2) << __func__ << ": Check NTP Images component update";
   BraveOnDemandUpdater::GetInstance()->OnDemandUpdate(component_id);
 }
 
+void NTPBackgroundImagesService::RegisterBackgroundImagesComponent() {
+  const auto& data = GetBackgroundImagesComponentData();
+
+  DVLOG(2) << __func__ << ": Start NTP BI component";
+  RegisterNTPBackgroundImagesComponent(
+      component_update_service_,
+      data->component_base64_public_key,
+      data->component_id,
+      base::StringPrintf("NTP Background Images"),
+      base::BindRepeating(&NTPBackgroundImagesService::OnComponentReady,
+                          weak_factory_.GetWeakPtr(),
+                          false, false));
+  // SI component checks update more frequently than other components.
+  // By default, browser check update status every 5 hours.
+  // However, this background interval is too long for SI. Use 1 hour interval.
+  si_update_check_timer_.Start(
+      FROM_HERE,
+      base::TimeDelta::FromHours(kSIComponentUpdateCheckIntervalHours),
```

Comment:
perhaps better to use default value for background images? Sponsored images are different, we want them live as fast as we can.

Reply:
Right, I think we don't need to have custom update interval for this.

Reply:
Fixed now.